csstools/index.html

```diff
@@ -786,7 +786,34 @@
      </section>
     </div>
     
-   
+
+    <div class="slide section"> 
+     <header><h1>Extras</h1></header> 
+     <section class="content"></section>
+    </div>
+    
+    <div class="slide normal"> 
+      <header><h1>Other things worth considering</h1></header> 
+      <section class="content"> 
+        <p>Performance</p>
+        <ul>
+          <li>CSS file size and compression
+          <li>Image sprites
+          <li>Data URI
+          <li>Icon fonts
+        </ul>
+        <p>CSS preprocessors</p>
+        <ul>
```

Comment:
I kind of think they won't understand what preprocessors are without the example. How about:
CSS workflow in large apps:
- Linters (CSSLint)
- Minifiers (YUI, CSMinifier)
- Preprocessors (Less, Sass)
- Build tools (Makefile, Grunt, Gulp)
with links to each.
Basically, they don't need to care about all those things unless they're making a webapp of a significant size, so I think that bucketing makes sense.